spec/index.bs

```diff
@@ -762,36 +769,74 @@ enum GPUCompareFunction {
 Resource Binding {#binding}
 ===========================
 
-## GPUPipelineLayout ## {#pipeline-layout}
+## GPUBindGroupLayout ## {#bind-group-layout}
+
+A {{GPUBindGroupLayout}} defines an interface between a set of resources bound in a {{GPUBindGroup}} and their accessibility in shader stages.
```

Comment:
nit: shouldn't this be "the interface"? Technically, it's a very concrete contract it is defining, not "any" contract.